<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Blender_Import_Dialog</class>
 <widget class="QDialog" name="Blender_Import_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import CSV</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nodes CSV</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Materials</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Bars_LEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bars CSV</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Forces CSV</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Forces_LEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Supports CSV</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Materials_LEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Supports_LEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Nodes_LEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="Nodes_Button">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Bars_Button">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Materials_Button">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Forces_Button">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Supports_Button">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="Ok_Button">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Nodes_Button</tabstop>
  <tabstop>Bars_Button</tabstop>
  <tabstop>Materials_Button</tabstop>
  <tabstop>Forces_Button</tabstop>
  <tabstop>Supports_Button</tabstop>
  <tabstop>Nodes_LEdit</tabstop>
  <tabstop>Bars_LEdit</tabstop>
  <tabstop>Materials_LEdit</tabstop>
  <tabstop>Forces_LEdit</tabstop>
  <tabstop>Supports_LEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Ok_Button</sender>
   <signal>accepted()</signal>
   <receiver>Blender_Import_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ok_Button</sender>
   <signal>rejected()</signal>
   <receiver>Blender_Import_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
